packit/config/package_config.py

```diff
@@ -254,6 +255,85 @@ def __eq__(self, other: object):
         )
 
 
+class PackageConfigValidation:
+    """
+    Package config validation
+    Does not validate fields: jobs
+    """
+
+    def __init__(self, package_config: PackageConfig):
+        self.package_config: PackageConfig = package_config
+
+    def validate(self) -> dict:
+        return {
+            **self.validate_primitive(),
+            **self.validate_synced_files(),
+            **self.validate_notifications(),
+        }
+
+    def validate_primitive(self) -> dict:
+        from packit.schema import PackageConfigSchema
+
+        try:
+            config = self.package_config
+            return PackageConfigSchema().validate(
+                {
+                    "config_file_path": config.config_file_path,
+                    "specfile_path": config.specfile_path,
+                    "downstream_package_name": config.downstream_package_name,
+                    "upstream_project_url": config.upstream_project_url,
+                    "upstream_package_name": config.upstream_package_name,
+                    "upstream_ref": config.upstream_ref,
+                    "upstream_tag_template": config.upstream_tag_template,
+                    "dist_git_base_url": config.dist_git_base_url,
+                    "dist_git_namespace": config.dist_git_namespace,
+                    "spec_source_id": config.spec_source_id,
+                    "create_pr": config.create_pr,
+                    "actions": config.actions,
+                    "create_tarball_command": config.create_tarball_command,
+                    "current_version_command": config.current_version_command,
+                    "allowed_gpg_keys": config.allowed_gpg_keys,
+                    "patch_generation_ignore_paths": config.patch_generation_ignore_paths,
+                }
```

Comment:
Is this the data to be validated? Shouldn't this be `self.package_config` instead?